Source/Resources/en.lproj/Localizable.strings

```diff
@@ -1,40 +1,41 @@
-"missingBillingFormFullName" = "Missing Name";
-"missingBillingFormAddressLine1" = "Missing Address Line 1";
-"missingBillingFormAddressLine2" = "Missing Address Line 2";
-"missingBillingFormCity" = "Missing City";
-"missingBillingFormCounty" = "Missing County";
-"missingBillingFormPostcode" = "Missing Postcode";
-"missingBillingFormPhoneNumber" = "Missing Phone Number";
-"missingBillingFormCountry" = "Missing Country";
-"billingFormPhoneNumberHint" = "Phone Number";
-"billingAddressTitle" = "Billing address";
-"BillingFormCountryButtonTitle" = "Please select a country";
-"PaymentFormSummaryViewHintText" = "We need this information as an additional security measure to verify this card.";
-"AddBillingAddress" = "Add billing address";
+"Missing Name" = "Missing Name";
+"Missing Address Line 1" = "Missing Address Line 1";
+"Missing Address Line 2" = "Missing Address Line 2";
+"Missing City" = "Missing City";
+"Missing County" = "Missing County";
+"Missing Postcode" = "Missing Postcode";
+"Missing Phone Number" = "Missing Phone Number";
+"Missing Country" = "Missing Country";
+"Please select a country" = "Please select a country";
+"We need this information as an additional security measure to verify this card." = "We need this information as an additional security measure to verify this card.";
+"Add billing address" = "Add billing address";
```

Comment:
This is actually really bad. The main reason localisation keys are using some other format than potentially their value is to make them standout at development time if not linked properly.

If this was automated it needs to be disabled (at least for file, or whatever level you may find its needed) and the changes undone!

Reply:
Question : @ehab-al-cko is the RW guide making us to this change ?. What do you think about this change. 

Especially on line 10 where it makes us right long keys which could be overkill. Let me know your thoughts. Again this one is a matter of agreeing between devs ( whatever you both decide I am happy with it)